src/sidebar/app.js

```diff
@@ -135,38 +135,41 @@ module.exports = angular.module('h', [
   // The root component for the application
   .directive('hypothesisApp', require('./directive/app'))
 
-  // UI components and helpers
-  .directive('annotation', require('./directive/annotation').directive)
-  .directive('annotationShareDialog', require('./directive/annotation-share-dialog'))
-  .directive('annotationThread', require('./directive/annotation-thread'))
-  .directive('dropdownMenuBtn', require('./directive/dropdown-menu-btn'))
+  // UI components
+  .component('annotation', require('./directive/annotation').component)
+  .component('annotationShareDialog', require('./directive/annotation-share-dialog'))
+  .component('annotationThread', require('./directive/annotation-thread'))
+  .component('dropdownMenuBtn', require('./directive/dropdown-menu-btn'))
+  .component('helpLink', require('./directive/help-link'))
+  .component('helpPanel', require('./directive/help-panel'))
+  .component('loggedoutMessage', require('./directive/loggedout-message'))
+  .component('loginControl', require('./directive/login-control'))
+  .component('loginForm', require('./directive/login-form').component)
+  .component('publishAnnotationBtn', require('./directive/publish-annotation-btn'))
+  .component('searchInput', require('./directive/search-input'))
+  .component('searchStatusBar', require('./directive/search-status-bar'))
+  .component('selectionTabs', require('./directive/selection-tabs'))
+  .component('sidebarTutorial', require('./directive/sidebar-tutorial').component)
+  .component('shareDialog', require('./directive/share-dialog'))
+  .component('sortDropdown', require('./directive/sort-dropdown'))
+  .component('svgIcon', require('./directive/svg-icon'))
+  .component('tagEditor', require('./directive/tag-editor'))
+  .component('threadList', require('./directive/thread-list'))
+  .component('timestamp', require('./directive/timestamp'))
+
+  // These should use `component()` but will require some changes.
+  .directive('groupList', require('./directive/group-list').directive)
+  .directive('markdown', require('./directive/markdown'))
+  .directive('topBar', require('./directive/top-bar'))
```

Comment:
So can the remaining directives below really _not_ become components, even with changes?